test/connection.test.ts

```diff
@@ -56,8 +56,8 @@ describe('Connection', function () {
     })
   })
 
-  describe('end', function () {
-    it('should close the other side of the connection', async function () {
+  describe('end', function() {
+    it('should close the other side of the connection', async function() {
```

Comment:
nit: standard style says there should be a space between function and the open paren